spec/controllers/invites_controller_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+describe InvitesController  do
+  describe "GET new" do
+    it "sets the @invite variable" do
+      user = Fabricate(:user)
+      set_current_user(user)
+      get :new
+      expect(assigns(:invite)).to be_instance_of(Invite)
+    end
+    it_behaves_like "requires sign in" do
+      let(:action) { get :new }
+    end
+  end
+
+  describe "POST create" do
+    it_behaves_like "requires sign in" do
+      let(:action) { post :create }
+    end
+    
+    context "with valid inputs" do
+      it "redirects to the home page" do
+        user = Fabricate(:user)
+        set_current_user(user)
+        post :create, invite: Fabricate.attributes_for(:invite)
+        expect(response).to redirect_to home_path
+      end
```

Comment:
There should be an empty space line between spec examples as it makes a bit more readable.